### PR TITLE
Integrate Epic EOS anti cheat

### PR DESCRIPTION
**Pull request by matiascalegaris**

### CODE REVIEW

File: Codigo/AntiCheat.bas [hunk 1]
```diff
@@ -0,0 +1,177 @@
+Attribute VB_Name = "AntiCheat"
```
Comment (morgolock) [inline Codigo/AntiCheat.bas:1]: Please add the licence header to any new files

**Review by morgolock — CHANGES_REQUESTED** [review]

File: Codigo/Protocol.bas [hunk 2]
```diff
@@ -10725,11 +10727,20 @@ HandleUseHKeySlot_Err:
...
+    Exit Sub
+AntiCheatMessage_Err:
+    Call TraceError(Err.Number, Err.Description, "Protocol.AntiCheatMessage", Erl)
+End Sub
+
 Public Sub HendleRequestLobbyList(ByVal UserIndex As Integer)
 On Error GoTo HendleRequestLobbyList_Err:
-    
     Call WriteUpdateLobbyList(UserIndex)
-    Exit Sub
```
Comment (morgolock) [inline Codigo/Protocol.bas:10732]: We should not remove `exit sub`

**Review by morgolock — CHANGES_REQUESTED** [review]
Is there no way to disable the Anti cheat code when compiling like in the client with a define?

**Comment by matiascalegaris** [comment]
> Is there no way to disable the Anti cheat code when compiling like in the client with a define?

the anti cheat is disabled by default under a featuretoggle, no need to build flag

**Pull request merged**